src/pke/include/cryptocontext.h

```diff
@@ -860,7 +860,7 @@ class CryptoContextImpl : public Serializable {
    */
   virtual Plaintext MakeCKKSPackedPlaintext(
```

Comment:
If slots = 0, we should set slots to the BatchSize (available through EncodingParams of Cryptocontext)

Reply:
Changing this. 
FYI: in CKKSPackedEncoding constructors "slots" is checked for 0 and assigned to GetElementRingDimension() / 2, if it is true.